File: Modul-4/Soal 3/PRAK403-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main() {
    int angka1;
    int angka2;
    printf("");
    scanf("%d %d", &angka1, &angka2);
    
    if(angka1<angka2){
        for(int i=angka1, j=angka2; i<=angka2 && j>=angka1; i++, j--){
            printf("%d %d", i, j);
            if(i<angka2 || j>angka1){
                printf(" - ");
            }
        }
    } else {
        for(int i=angka1, j=angka2; i>=angka2 && j<=angka1; i--, j++){
            printf("%d %d", i, j);
            if(i>angka2 || j<angka1){
                printf(" - ");
            }
        }
    }

    return 0;
}

File: Modul-4/Soal 3/PRAK403-2410817120019-NadyaRamadhani.py
angka1, angka2 = map(int, input("").split())

if angka1<angka2:
    for i, j in zip(range(angka1, angka2+1), range(angka2, angka1-1, -1)):
        print(f"{i} {j}", end=' ')
        if i<angka2 or j>angka1:
            print("-", end=' ')
else:
    for i, j in zip(range(angka1, angka2-1, -1), range(angka2, angka1+1)):
        print(f"{i} {j}", end=' ')
        if i>angka2 or j<angka1:
            print("-", end=' ')